.travis.yml

```diff
@@ -6,7 +6,7 @@ install:
 script: make tox
 env:
   - TOXENV=py27
-  - TOXENV=pypy
+  - TOXENV=py36
```

Comment:
The modifications will make `pypy` failed? Otherwise, maybe we can just add one more entry without removing `pypy`